### DISCUSSION TITLE
About Open MCT Discussions

### DISCUSSION THREAD

**Discussion by akhenry** [nasa/openmct#3750] [Announcements]
# Welcome to Open MCT's Discussions

### [Announcements](https://github.com/nasa/openmct/discussions/categories/announcements)

The Open MCT team will use this discussion category for any news or information about Open MCT.

Some examples:
  * General Updates
  * Build Releases
  * Deprecations
  * Anything else we think users of Open MCT should know

**This section will ONLY be used by the Open MCT team, please do not make any posts in this category as they will be deleted.**

***

### [Community Q&A](https://github.com/nasa/openmct/discussions/categories/community-q-a)

This category is for asking questions... and answering them! Chances are if you've encountered a certain issue or have a certain question, someone else has as well. Open MCT has an awesome community, let's help each other out!

Also, remember to check out our [API documentation](https://nasa.github.io/openmct/documentation/) and our [plugins](https://nasa.github.io/openmct/plugins/). Your answer may be in there!

Also, if you've found a bug, don't hesitate to [raise an issue](https://github.com/nasa/openmct/issues/new) and let us know!

Some examples of Q&A topics:
  * Can someone help me to get the [_cool plugin_] plugin working?
  * Will Open MCT add [_cool feature_] in the future?
  * What's your favorite Open MCT plugin?
  * Does anyone know of a plugin that does [_cool thing_]?

**This section is open to the community, ask away! (_and answer away!_)**

***

### [Show & Tell](https://github.com/nasa/openmct/discussions/categories/show-and-tell)

You made a really cool plugin didn't you? Did you implement Open MCT in a totally unique way? Well, this category is for you! If you've got a cool Open MCT addition or variation or you just want to see what other cool stuff people have come up with, this is your spot.

Some examples:
  * Check out this amazing plugin, it does [_something really cool, obviously_]!
  * Look! I built a view using [_some other front-end framework_]...
  * My implementation of Open MCT on [_really cool architecture setup_].
  * My custom [_super dope sounding name_] theme for Open MCT

**This section is open to the community, show us what you got! (_or just see other people's super cool stuff..._)**

***

**Comment by aimanafzal** [comment]
Hi @akhenry, I sent a signed copy of NDA to get started in contributing to OpenMCT a few months back. 
Still looking forward to a Go Ahead. 
Any additional steps required to start contributing in OpenMCT?
